spangres/toddl.go

```diff
@@ -0,0 +1,239 @@
+// Copyright 2021 Google LLC
```

Comment:
I don't feel too strongly, but this is a lot of code to duplicate for the few lines of changes we need to Spangres.

The other option would just be to put to duplicate the code and just have an extra parameter (say SpannerDialect?) which we use to guard the relevant code modifications in a shared file for toddl that covers both Spanner and Spangres.